src/app/toolbar/toolbar.component.html

```diff
@@ -1,17 +1,22 @@
 <div class="toolbar" role="banner">
-  <img
-    alt="Pokemon Logo"
-    src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/98/International_Pok%C3%A9mon_logo.svg/2560px-International_Pok%C3%A9mon_logo.svg.png"
-  />
+  <a id="logoLink" href="/">
+    <img
+      alt="Pokemon Logo"
+      src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/98/International_Pok%C3%A9mon_logo.svg/2560px-International_Pok%C3%A9mon_logo.svg.png"
+    />
+  </a>
   <div>
-    <a
+    <a id="homeLink" href="/"> HOME </a>
+
+    <!-- leave this comments for learning purposes -->
```

Comment:
What did we learn here?

Reply:
<img width="340" alt="confused cave man clean" src="https://user-images.githubusercontent.com/1997886/127285308-15402465-e5db-4371-bafe-099d52418651.png">